CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Getting Started with Contributing
+
+
+**Step 1:** Fork the repository
+
+**Step 2:** Clone the forked repository to your local machine with 
+```
+git clone https://github.com/postmanlabs/newman-dashboard

```

Comment:
@soham4abc can you just check if https cloning of git repositories is across git versions or specific to some?

Reply:
https cloning causes problems in Git versions older than 1.7.10.
Should I add some more documentation to update git to latest version?